content/dstu2/document-reference.md

```diff
@@ -13,7 +13,7 @@ title: DocumentReference | DSTU 2 API
 
 ## Create
 
-Create new documents. Only documents that would be classified as notes are currently supported.
+Create new documents. Currently only XHTML formatted documents are supported. Only documents that need to be associated to the patient's chart should be written. Please validate your document is valid XHTML using any available validator like the one linked [here]. 
```

Comment:
Why remove the comment about notes? This is currently a limitation of the service - it draws on the freetext note in the underlying code, I believe.